Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -49,13 +52,16 @@ There are usually two release engineers working at any given time. It goes witho
   1. On the sidebar click `+ Version or Platform` and select `iOS`.
   1. Input the new version number.
 
-### Phase 2: Test and fix bugs
+### Phase 3: Test and fix bugs
 *It starts after the App Center build has been delivered and it can take several cycles*
 
 1. Testers will then begin their work against the build you just created.
-1. Any hotfix should target that branch, and you, as the release engineer, are responsible for double checking that the hotfix's PR is pointing to the release branch (instead of `develop`). The issue for the hotfix should be added to the release JIRA board.
+1. Any hotfix should target the `develop` branch first. These PRs need to be reviewed not just by the relevant squad or platform QA (Bibin Paul), but also by one of the release engineers assigned to the release. This is to ensure visibility of changes being requested to be made, and to ensure the correct builds are available for validation.
```

Comment:
Is there a link to a page which keeps current platform QA up-to-date? Instead of using Bibin Paul's name directly here.

Reply:
Hmm no idea if we've such a page. In any case the document already has Andreea and Lilla's names there, explicitely stated.